### ISSUE TITLE
Figure out how to handle a failed pipeline run^

### ISSUE THREAD

**Issue by danjoneslf** [alphagov/cyber-security-windows-sandbox#37]
At the moment if the pipeline stops then the sandbox doesn't get destroyed. 
If the sandbox doesn't get destroyed then 
a) it's costing us money 
b) it's an unmaintained windows server and 
c) the next time you run the pipeline it 
  i) creates a new keypair and updates secrets manager to overwrite the current values 
  ii) then does an apply over the existing deployed TF which doesn't do anything 
  iii) fails the smoke test because the events were created too long ago 
  iv) fails to decode the Administrator password with the new data and overwrites it in SSM with "missing" 

Basically once that's happened you have to manually run a `destroy` to recover it.

**Comment by danjoneslf** [comment]
This will be fixed by splunk-apps PR [675](https://github.com/alphagov/cyber-security-splunk-apps/pull/675).

**Comment by danjoneslf** [comment]
This is resolved by the above Splunk apps PR.
